client/package.json

```diff
@@ -51,6 +51,7 @@
   "dependencies": {
     "bootstrap": "^3.3.7",
     "history": "^4.2.0",
+    "jquery": "^3.1.1",
```

Comment:
We are already using React to control DOM. I don't see why we would need jQuery.